src/Media/ImageCollectionFactory.php

```diff
@@ -47,12 +50,16 @@ public function withUuidRegex($mediaIdentifierRegex)
     }
 
     /**
-     * @inheritdoc
+     * @param CultureFeed_Cdb_Data_Media $media
+     * @param Description $fallbackDescription,
+     * @param CopyrightHolder $fallbackCopyright
+     * @return ImageCollection
      */
-    public function fromUdb2Media(
+    private function fromUdb2Media(
```

Comment:
👍 

Kunnen we de `normalize()` method op deze class ook private maken of wordt deze nog ergens extern gebruikt? Is alleszins geen onderdeel van de interface.

Eventueel de private methods dan ook groeperen vanonder in de class. Conventie is om altijd eerst public methods te definiëren, daarna protected en als laatste de private.